trino/integration_test.go

```diff
@@ -106,6 +113,8 @@ func TestMain(m *testing.M) {
 			log.Fatalf("Timed out waiting for container to get ready: %s", err)
 		}
 		*integrationServerFlag = "http://test@localhost:" + resource.GetPort("8080/tcp")
+
+		http.DefaultTransport.(*http.Transport).TLSClientConfig = &tls.Config{InsecureSkipVerify: true}
```

Comment:
Can we test without `InsecureSkipVerify`?

Reply:
@nineinchnick - We can not easily test without InsecureSkipVerify as I have self signed certificate. The process of verifying certificate is more complicated, I think not worth for integration testing.

Reply:
@nineinchnick - Still working on this. I will have this by EOD.